api/config.go

```diff
@@ -52,6 +52,10 @@ type ConfigHookRequest struct {
 	App  string `json:"receive_repo"`
 }
 
+// Healthchecks is a map of different Healthcheck's that a process of application uses.
```

Comment:
Typo: should be "Healthchecks" without an apostrophe `'`. Also I think this should be reworded a bit:
> // Healthchecks maps an app process name to a healthcheck type.
> // The valid values are "livenessProbe" and "readinessProbe".